interface/CH.json

```diff
@@ -1652,10 +1422,10 @@
         }
       }
     },
-    "/BrugervendteSystemer/{id}/SamlMetadata": {
+    "/BrugervendteSystemer/{id}/OIOSAMLProfile/{value}": {
       "put": {
-        "summary": "Add a SAML metadata version on the specified user system",
-        "description": "Add a `SamlMetadata` version on the specified `BrugervendtSystem`",
+        "summary": "Set the OIOSAMLProfile for this BrugervendtSystem (user system)",
+        "description": "Set the 'OIOSAMLProfile' for this `BrugervendtSystem` (user system)",
```

Comment:
'OIOSAMLProfile' -> should this one be `OIOSAMLProfile`? Many other places have the same mistake as well

Reply:
fixed